fastly/fixtures/services/details.yaml

```diff
@@ -1,46 +1,44 @@
TK9wS159v2gPAZ8A","locked":false,"created_at":"2021-01-13T19:00:33Z","testing":false,"active":false,"staging":false,"deleted_at":null,"comment":"","deployed":false},{"number":5,"updated_at":"2021-01-13T19:00:36Z","service_id":"7i6HN3TK9wS159v2gPAZ8A","locked":false,"testing":false,"created_at":"2021-01-13T19:00:33Z","deleted_at":null,"comment":"","staging":false,"deployed":false,"active":false},{"locked":false,"service_id":"7i6HN3TK9wS159v2gPAZ8A","number":6,"updated_at":"2021-01-13T19:00:37Z","active":false,"deleted_at":null,"staging":false,"comment":"","deployed":false,"created_at":"2021-01-13T19:00:34Z","testing":false},{"service_id":"7i6HN3TK9wS159v2gPAZ8A","locked":false,"number":7,"updated_at":"2021-01-13T19:00:39Z","comment":"","staging":false,"deleted_at":null,"deployed":false,"active":false,"testing":false,"created_at":"2021-01-13T19:00:34Z"},{"created_at":"2021-01-13T19:00:34Z","testing":false,"service_id":"7i6HN3TK9wS159v2gPAZ8A","active":false,"deployed":false,"deleted_at":null,"comment":"","staging":false,"updated_at":"2021-01-13T19:00:37Z","number":8,"locked":false},{"locked":false,"service_id":"7i6HN3TK9wS159v2gPAZ8A","updated_at":"2021-01-13T19:00:39Z","number":9,"deployed":false,"deleted_at":null,"comment":"","staging":false,"active":false,"created_at":"2021-01-13T19:00:35Z","testing":false},{"created_at":"2021-01-13T19:00:35Z","testing":false,"deployed":false,"deleted_at":null,"staging":false,"comment":"","active":false,"service_id":"7i6HN3TK9wS159v2gPAZ8A","updated_at":"2021-01-13T19:00:38Z","number":10,"locked":false},{"testing":false,"created_at":"2021-01-13T19:00:36Z","service_id":"7i6HN3TK9wS159v2gPAZ8A","active":false,"deleted_at":null,"comment":"","staging":false,"deployed":false,"number":11,"updated_at":"2021-01-13T19:00:38Z","locked":false},{"created_at":"2021-01-13T19:00:36Z","testing":false,"service_id":"7i6HN3TK9wS159v2gPAZ8A","active":false,"staging":false,"comment":"","deleted_at":null,"deployed":false,"number":12,"updated_at":"2021-01-13T19:00:39Z","locked":false},{"testing":false,"created_at":"2021-01-13T19:00:37Z","service_id":"7i6HN3TK9wS159v2gPAZ8A","active":false,"deployed":false,"comment":"","deleted_at":null,"staging":false,"updated_at":"2021-01-13T19:00:39Z","number":13,"locked":false},{"number":14,"updated_at":"2021-01-13T19:00:39Z","locked":false,"service_id":"7i6HN3TK9wS159v2gPAZ8A","testing":false,"created_at":"2021-01-13T19:00:37Z","deleted_at":null,"comment":"","staging":false,"deployed":false,"active":false},{"locked":false,"service_id":"7i6HN3TK9wS159v2gPAZ8A","number":15,"updated_at":"2021-01-13T19:00:38Z","active":false,"comment":"","staging":false,"deleted_at":null,"deployed":false,"created_at":"2021-01-13T19:00:37Z","testing":false},{"number":16,"updated_at":"2021-01-13T19:00:38Z","service_id":"7i6HN3TK9wS159v2gPAZ8A","locked":false,"created_at":"2021-01-13T19:00:38Z","testing":false,"staging":false,"deleted_at":null,"comment":"","deployed":false,"active":false},{"deployed":false,"comment":"new
```

Comment:
@ovv would you mind replacing `integralist-test-go-fastly-tests.com` in the below fixture output with `test-service.com`. It's not a concern it being there, but would be better if my username wasn't in the fixture data really 🙂 

This shouldn't break any tests as nothing is validating the domain AFAIK.

Thanks!